<?xml version="1.0"?>

<launch>
	<node name="odom_noise" type="odom_noise" pkg="evolutive_map" output="screen">
		<param name="alpha1" value="0.001" type="double" />
		<param name="alpha2" value="0.001" type="double" />
		<param name="alpha3" value="0.005" type="double" />
		<param name="alpha4" value="0.005" type="double" />
	</node>

    <node name="calibrate_odom_noise" type="calibrate_odom_noise" pkg="evolutive_map" output="screen" />
</launch>
